python_modules/dagster/dagster/check/__init__.py

```diff
@@ -317,14 +317,19 @@ def is_list(obj_list: Any, of_type: Type = None, desc: str = None) -> List:
     return _check_list_items(obj_list, of_type)
 
 
-def is_tuple(obj_tuple: Any, of_type: Type = None, desc: str = None) -> Tuple:
+def is_tuple(
+    obj_tuple: Any,
+    of_type: Type = None,
+    of_shape: Optional[Tuple[Type, ...]] = None,
+    desc: str = None,
+) -> Tuple:
```

Comment:
can you add doc block here for what exactly these args mean? 
is it valid to set of type and of shape? I dont see an error raised for both passed

Reply:
Good point, I added docs and check for simultaneous of_type/op_shape